app/cdap/components/hydrator/components/SidePanel/ChangeVersionMenu.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState, useRef, MouseEvent } from 'react';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import EditIcon from '@material-ui/icons/Edit';
+// import styled from 'styled-components';
+
+import { ToolTipButtonContainer } from './sharedStyled';
+import Typography from '@material-ui/core/Typography';
+
+interface IChangeVersionMenuProps {
+  changePluginVersion: (plugin) => void;
+  plugin: any;
+}
+/**
+ * How the angular version of this is super weird - it uses the same click function for all
+ * kinds of things including delete, create, add to page etc. It also passes around contentData as
+ * a plugin and things like that. I believe it also automatically sends a request to change the default version
+ * when you change a version. It is very confusing and hard to use. Most of the code is in leftpanel-ctrl and group-side-panel
```

Comment:
Please double-check line length in this comment

Reply:
what length do we want to limit it to? 120? 80? 150?